### ISSUE TITLE
[TF2] Request: Ability to Kick a Player from a Competitive Match

### ISSUE THREAD

**Issue by SanAntonio2** [ValveSoftware/Source-1-Games#4282]
The main reason is cheaters.
The game doesn't provide an opportunity to deal with such people, so you have to either finish the game or leave the match and get a ban.

**Comment by condor00fr** [comment]
Yeah, it's annoying that the game is basicly lost because one dude thought it would be funny to go noclip and full crite on our a#ses. another thing that makes people go livid is the fact that for some reason the game is considered cause a guy leaves the lobby. Like.... why?

**Comment by rabscootle** [comment]
Restoring the in-game report tool to its old (functional) state is probably a much more effective way to curb cheaters in Competitive.

**Comment by condor00fr** [comment]
It dosn't work?

**Comment by SanAntonio2** [comment]
> It dosn't work?

In competitive mode, you can’t kick players at all


> Yeah, it's annoying that the game is basicly lost because some random dude thought it would be funny to go noclip and full crits on our a#ses. another thing that makes people go livid is the fact that for some reason the game consider the match lost cause a guy leaves the lobby. Like.... why?

I have not seen anything like this, but I have seen snipers with aimbots. Also, about a year ago, I came across people who did not allow the match to start. That is, the countdown began, before the start of the game, this player, in the last seconds, left the match, which canceled the countdown and the game didn't start


> Restoring the in-game report tool to its old (functional) state is probably a much more effective way to curb cheaters in Competitive.

Look at it from several angles:
- You have met against a cheater, the enemy team cannot kick him out, you have to wait until the game is over (either stand AFK or try to resist) or leave during the match so as not to waste your time, after which you accordingly receive a game ban (You won't get MMR for playing anyway).
- Now you can kick players in competitive mode, instead of a cheater, a bot will play (this does not affect the receipt of MMR in any way).
- You can't kick a player, but the reporting system has been updated, every day players send several hundred reports on cheaters and innocent players, the Valve has to manually check each report. (You don't get MMR, the whole match you have to put up with a cheater)

**Comment by condor00fr** [comment]
Ok nice to know,.thanks

**Comment by PhillipoTF2** [comment]
The whole of the Competitive Mode needs to be redone. Vote kicking can only really become a reality once the ability to join in-progress games is a thing. Although this would not be needed if the in-game report system actually did something.

Doubt this will get much attention though. I think valve just gave up on Competitive Mode as there is so much that needs to be changed with it for the mode to work.

**Comment by gaelcoral** [comment]
Please I need the ability to Kick, that's why I stopped playing competitive mode.
